MonoGame.Framework.Content.Pipeline/Graphics/GraphicsUtil.cs

```diff
@@ -159,7 +167,9 @@ public static void CompressDxt(GraphicsProfile profile, TextureContent content,
 
             if (alphaRange == AlphaRange.Opaque)
                 Compress(typeof(Dxt1BitmapContent), content, generateMipMaps);
-            else if (alphaRange == AlphaRange.Cutout || sharpAlpha)
+            else if (sharpAlpha)
+                CompressFontDXT3(content, generateMipMaps);
```

Comment:
I guess isSpriteFont is best as 16bit color isn't necessarily a good thing for just any greyscale texture.